src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceSpecCallbackHandler.java

```diff
@@ -301,18 +314,19 @@ private CallbackResponse populateCaseData(CallbackParams callbackParams) {
         var caseData = callbackParams.getCaseData();
 
         CaseData.CaseDataBuilder<?, ?> updatedCaseData = caseData.toBuilder();
+        boolean notVersion0 = EnumSet.of(V_1, V_2).contains(callbackParams.getVersion());
```

Comment:
Sorry for my code taste we could create a name more readable because `noVersion0` isn't reflect that the command do.

Reply:
Changed to hasVersion